src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/SqlBulkCopyTest.cs

```diff
@@ -247,5 +247,43 @@ public void DestinationTableNameWithSpecialCharTest()
         {
             DestinationTableNameWithSpecialChar.Test(srcConstr, AddGuid("SqlBulkCopyTest_DestinationTableNameWithSpecialChar"));
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void OrderHintTest()
+        {
+            OrderHint.Test(srcConstr, dstConstr, AddGuid("SqlBulkCopyTest_OrderHint"));
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void OrderHintMissingTargetColumnTest()
+        {
+            OrderHintMissingTargetColumn.Test(srcConstr, dstConstr, AddGuid("SqlBulkCopyTest_OrderHintMissingTargetColumn"));
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void OrderHintDuplicateColumnTest()
+        {
+            OrderHintDuplicateColumn.Test(srcConstr, dstConstr, AddGuid("SqlBulkCopyTest_OrderHintDuplicateColumn"));
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void OrderHintTransactionTest()
+        {
+            OrderHintTransaction.Test(srcConstr, dstConstr, AddGuid("SqlBulkCopyTest_OrderHintTransaction"));
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void OrderHintAsyncTest()
+        {
+            OrderHintAsync.Test(srcConstr, dstConstr, AddGuid("SqlBulkCopyTest_OrderHintAsync"));
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        [ActiveIssue(1)]
```

Comment:
We put the related task number from the task list inside the parentheses if it is defined.